demo/src/androidTest/java/com/novoda/movies/ProgrammaticallyCreatedViewTest.java

```diff
@@ -0,0 +1,49 @@
+package com.novoda.movies;
+
+import android.content.Context;
+import android.support.test.filters.LargeTest;
+import android.support.test.runner.AndroidJUnit4;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.novoda.espresso.ViewCreator;
+import com.novoda.espresso.ViewTestRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static android.view.ViewGroup.LayoutParams.WRAP_CONTENT;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class ProgrammaticallyCreatedViewTest {
+
+    private static final String TEXT = "Hello world!";
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
```

Comment:
doesn't look like the this rule is needed?

Reply:
commit 6b56803bcdb0baaf33c1ef4fc4e1aa181c68a397
Author: Ataul Munim <hello@ataulm.com>
Date:   Fri Nov 10 09:33:37 2017 +0000

    Remove unnecessary mockito rule